src/libmongoc/src/mongoc/mongoc-write-command.c

```diff
@@ -59,6 +60,7 @@ _mongoc_write_command_insert_append (mongoc_write_command_t *command, const bson
       bson_concat (&tmp, document);
       _mongoc_buffer_append (&command->payload, bson_get_data (&tmp), tmp.len);
       bson_destroy (&tmp);
+
```

Comment:
```suggestion
```

Remove unneeded whitespace change to reduce diff. That may reduce noise when reviewing git history.